sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -3781,6 +3781,14 @@ object SQLConf {
     .intConf
     .createWithDefault(0)
 
+  val PLANNED_WRITE_ENABLED = buildConf("spark.sql.plannedWrite.enabled")
+    .internal()
+    .doc("When set to true, Spark adds logical sorts to V1 write commands if needed so that " +
```

Comment:
`Spark` -> `Spark optimizer` could be clearer that sort is added during query planning.